### ISSUE TITLE
No project.clj

### ISSUE THREAD

**Issue by alanmarazzi** [jtwool/WonderfulCoyote#1]
To make code reproducible it would be nice to include a `project.clj`

**Comment by jtwool** [comment]
You're right—I'm starting to add the key `project.clj` information in the headers of the files as comments. Is there a specific file you'd like to run but can't?

**Comment by alanmarazzi** [comment]
Thanks! No, I was able to run the file, but if I was coming a month ago
(when my Clojure knowledge was limited) I would have had issues

Il gio 5 lug 2018, 13:22 jtwool <notifications@github.com> ha scritto:
